test_model_configs/single_population_node_with_updaters_config.txt

```diff
@@ -0,0 +1,732 @@
+index: 0
+name: "root"
+branch_node {
+  branches {
+    condition {
+      name: "cookie vs. cookieless. name field is optional."
+      op: EQUAL
+      value: "cookied"
+    }
+    node {
+        name: "cookied"
+        index: 2
+        branch_node {
+          branches {
+            condition {
+              name: "pool"
+              op: EQUAL
+              value: "cookied_pool"
+            }
+            node {
+              name: "cookied_pool"
+              index: 0
+                branch_node {
+                  branches {
+                    condition {
+                      name: "country"
+                      op: EQUAL
+                      value: "GB"
+                    }
+                    node {
+                      name: "cookied_pool-GB"
+                      index: 0
+                      branch_node {
+                        branches {
+                          condition {
+                            name: "acting_user_space"
+                            op: EQUAL
+                            value: "phone"
+                          }
+                          node {
+                            name: "cookied_pool-GB-phone"
+                            index: 0
+                            branch_node {
+                              branches {
+                                condition {
```

Comment:
This is correct if we select children by demo. But here it has only one child, so it should use op: TRUE.

Reply:
Shouldn't this be AND, because we have 3 subfilters below this?

<img width="546" alt="Screen Shot 2021-06-03 at 2 20 58 PM" src="https://user-images.githubusercontent.com/65198074/120693376-ec9f9b00-c476-11eb-8e93-d7ec4801423e.png">